src/resources/postcss/components/_search.pcss

```diff
@@ -48,18 +48,19 @@
 	.tribe-common-c-search {
 		@media (--viewport-medium) {
 			display: flex;
+			flex: 1 1 100%;
```

Comment:
has the markup changed such that this is a flex child? curious why we're adding this here.